c2corg_ui/externs/bootstrap-slider.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @fileoverview Externs for bootstrap-slider
+ *
+ * @externs
+ */
+
+
+/**
+ * @constructor
+ */
+function Slider(arg1, arg2) {};
+
+
+/**
+ * @param {Object} arg
```

Comment:
> Do we really want to spend that much time writing a typedef here?

No, I'd say we keep it to the minimum and only add the options that we actually use.